packages/action-menu/test/benchmark/test-lazy.ts

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import '@spectrum-web-components/action-menu/sp-action-menu.js';
+import { html, render } from '@spectrum-web-components/base';
+import {
+    removeSlottableRequest,
+    type SlottableRequestEvent,
+} from '@spectrum-web-components/overlay/src/slottable-request-event.js';
+import { measureFixtureCreation } from '../../../../test/benchmark/helpers.js';
+
+const handleSlottableRequest = (event: SlottableRequestEvent): void => {
+    import('@spectrum-web-components/menu/sp-menu-item.js');
```

Comment:
should we await for these imports to download first and allow the render happen once it is done?

Reply:
I agree it does seem strange not to `await` any dynamic import.

Reply:
This will be included in https://github.com/orgs/adobe/projects/50/views/2?pane=issue&itemId=53607748